fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1450,6 +1450,7 @@ void SimulationConfig::setStorageEngine(const TestConfig& testConfig) {
 	// Using [0, 4) to disable the RocksDB storage engine.
 	// TODO: Figure out what is broken with the RocksDB engine in simulation.
 	int storage_engine_type = deterministicRandom()->randomInt(0, 6);
+	storage_engine_type = 4;
```

Comment:
The comment says 4 is exclusive yet we are setting storage_engine_type to 4 here ... and it overrides the line above. This temporary? Otherwise, bit of cleanup?

Reply:
Just forcing the storage_engine_type to rocksdb  to run more test coverage.. This is for just testing. You can ignore this.Sorry!